plugin/storage/cassandra/schema/docker.sh

```diff
@@ -3,7 +3,7 @@
 # This script is used in the Docker image jaegertracing/jaeger-cassandra-schema
 # that allows installing Jaeger keyspace and schema without installing cqlsh.
 
-CQLSH=/opt/apache-cassandra-3.0.12/bin/cqlsh
+CQLSH=/usr/bin/cqlsh
```

Comment:
I would also propose renaming it. `docker.sh` is vague. What it does actually with docker? I could use this script with `ccm` like `CQLSH="ccm node1 cqlsh" docker.sh`